rxp-hpp-android/build.gradle

```diff
@@ -1,14 +1,14 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion 29
+    buildToolsVersion "29.0.0"
 
     defaultConfig {
         minSdkVersion 19
-        targetSdkVersion 23

```

Comment:
Do changing the `compileSdkVersion` and `targetSdkVersion` values have any effect on integrators leveraging this library?

Reply:
Now library will support to latest android version i.e (API level 29= Android Q). It will not effect to integrators because we are supporting to minSdkVersion 19 i.e Android 4.4 Kitkat.  Applications which are targeting to latest android versions will get benefit of it.